src/pages/workspace/taxes/WorkspaceTaxesPage.tsx

```diff
@@ -47,12 +49,16 @@ function WorkspaceTaxesPage({
     const styles = useThemeStyles();
     const theme = useTheme();
     const {translate} = useLocalize();
+    const {environmentURL} = useEnvironment();
     const [selectedTaxesIDs, setSelectedTaxesIDs] = useState<string[]>([]);
     const [isDeleteModalVisible, setIsDeleteModalVisible] = useState(false);
     const defaultExternalID = policy?.taxRates?.defaultExternalID;
     const foreignTaxDefault = policy?.taxRates?.foreignTaxDefault;
     const isFocused = useIsFocused();
 
+    const isConnectedToAccounting = Object.keys(policy?.connections ?? {}).length > 0;
+    const isConnectedToQbo = policy?.connections?.quickbooksOnline;
```

Comment:
Sorry for chiming in. But I think we should wrap it up with Boolean or use double negation operator here.
```suggestion
    const isConnectedToQbo = Boolean(policy?.connections?.quickbooksOnline);
```

Reply:
Done.